pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+7. Security Implications - If there are security concerns in relation to the PHEP, those concerns should be explicitly written out to make sure reviewers of the PHEP are aware of them.
+8. How to Teach This - For a PHEP that adds new functionality or changes behavior, it is helpful to include a section on how to teach users, new and experienced, how to apply the PHEP to their work.
+
+    This section may include key points and recommended documentation changes that would help users adopt a new feature or migrate their code to use a language change.
+
+9. Reference Implementation - The reference implementation must be completed before any PHEP is given status "Final", but it need not be completed before the PHEP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
+
+    The final implementation must include test code and documentation appropriate for the relevant PyHC project(s).
+
+10. Rejected Ideas - Throughout the discussion of a PHEP, various ideas will be proposed which are not accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the PHEP as well as preventing people from bringing up the same rejected idea again in subsequent discussions.
+
+    In a way this section can be thought of as a breakout section of the Rationale section that is focused specifically on why certain ideas were not ultimately pursued.
+
+11. Open Issues - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete and reduces people duplicating prior discussion.
+12. Footnotes - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+13. Copyright/license - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/) (see this PHEP for an example).
+
+# PHEP Formats and Templates
+PHEPs are UTF-8 encoded text files using the Markdown format. Markdown allows for rich markup that is still quite easy to read, but also results in good-looking and functional HTML. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
```

Comment:
I am open to the idea of requiring a template PHEP before this one is merged (e.g. I can write PHEP-2 once the format is agreed upon here but before approving PHEP-1).

Reply:
+1 to not using GitHub extensions (although if I'm being honest, I might even have to double check myself on that sometimes, since 90% of the markdown I write is GitHub READMEs).

I'm in favor of templates. People like them and it lowers barrier to entry.

Reply:
Okay, I will take an action to draft a template PHEP once it seems like we're converging on this.

Reply:
Thank you!

Reply:
Template is in #25